src/main/java/com/upversionlab/controller/CompanyController.java

```diff
@@ -14,15 +15,16 @@
 @RequestMapping("/companies")
 public class CompanyController {
     private Map<Integer, Company> companies = new HashMap<>();
+    private static final String COMPANY_RESOURCE_NAME = Company.class.getSimpleName();
```

Comment:
Constants should come first (and be probably be separated by an empty line from other attributes).